### ISSUE TITLE
Bug: PrimeFaces SelectOneMenu not parsing properly.

### ISSUE THREAD

**Issue by melloware** [primefaces-extensions/faces-intellisense#29]
Taglib XML:

```xml
 <description>
   <![CDATA[SelectOneMenu is an extended version of the standard Faces SelectOneMenu.]]>
</description>
```

JSON output:

```js
 {
        "name": "selectOneMenu",
        "description": "\n            \n        ",
}
```

**Comment by melloware** [comment]
@FlipWarthog duh i know why: https://github.com/primefaces/primefaces/blob/321ca1819cd18d91e12c5ec944016c5494bf949a/primefaces/src/main/resources/META-INF/primefaces.taglib.xml#L24772

I fixed it in 15.0.0 but the 14.0.0 tag libary still has a blank.

**Comment by FlipWarthog** [comment]
> @FlipWarthog duh i know why: https://github.com/primefaces/primefaces/blob/321ca1819cd18d91e12c5ec944016c5494bf949a/primefaces/src/main/resources/META-INF/primefaces.taglib.xml#L24772
> 
> I fixed it in 15.0.0 but the 14.0.0 tag libary still has a blank.

What about f:selectItem?

**Comment by melloware** [comment]
Missing in 2.3 JSF I just added it but I can see its there in the 4.0 and 5.0 taglibs for Faces.  2.3 is not getting any more updates so I manually fixed it.
